Update how to manage bookmarks

kulsoomzahra opened the pull request:
Added an image to help users easily perform bookmark actions like import, export, restore and backup.

gusgustavo reviewed (changes requested):
Hi,

1. Please remove the images that you added.
2. Review the path to add bookmarks. You removed the instructions, and added something about clicking on "Library icon" in the toolbar... that is not the default UI in Tor Browser.
3. After fixing these issues, please squash your commits or submit a new PR.

kulsoomzahra commented:
Hey @gusgustavo ,

1.  I've removed the images.
2.  Restored the instructions and also removed the "Library icon" which was not in the default UI.
3. I'm unsure what is needed to be done next in reviewing the path to add bookmarks.

 Can you please highlight if i missed out something?
